libc-test/semver/netbsd.txt

```diff
@@ -1085,6 +1085,10 @@ UT_NAMESIZE
 VDISCARD
 VDSUSP
 VLNEXT
+VM_INHERIT_COPY
+VM_INHERIT_DONATE_COPY
+VM_INHERIT_NONE
+VM_INHERIT_SHARE
```

Comment:
The non-Apple BSDs seem to use `MAP_` as the prefix rather than `VM_`. I think you'll need to add those constants and update these files.